migration/20180921-recalculate-work.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+import os
+import sys
+import logging
+from pdb import set_trace
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..")
```

Comment:
The package_dir had an extra ".."  -- I think it was copied from a migration script in core.